features/step_definitions/cli.rb

```diff
@@ -111,17 +111,17 @@
 # 1. download file from URL
 # 2. load it as an ERB file with the cucumber scenario variables binding
 # 3. runs `oc create` command over the resulting file
-When /^I run oc create( as admin)? over ERB URL: #{HTTP_URL}$/ do |admin, url|
-  step %Q|I download a file from "#{url}"|
-
+When /^I run oc create( as admin)? over ERB test file: #{QUOTED}$/ do |admin, file_path|
```

Comment:
In the old version we didn't have the quotes around the file. I think we should keep the convention. Which is use quotes if whatever thing is in the middle of step or use non-quoted in case that is last argument. Here we have `step name: param` this we do not quote the `param`. Instead of quoted, pls use `(.*)`.

Reply:
ok, updated... PTAL